src/goImport.ts

```diff
@@ -104,3 +107,52 @@ export function addImport(arg: string) {
 		}
 	});
 }
+
+export function addImportToWorkspace(arg: string) {
+	const editor = vscode.window.activeTextEditor;
+	const selection = editor.selection;
+
+	let importPath = '';
+	if (selection.isSingleLine) {
+		// Attempt to load a partial import path based on currently selected text
```

Comment:
Can you provide an example for the case that you are attempting to cover here?

Reply:
Absolutely. The easiest example that comes to mind.

`import "github.com/aws/aws-sdk-go/service/rds/rdsutils"`

Realistically being able to select `github.com/aws/aws-sdk-go` and add that to the workspace is probably going to be what you want more than `github.com/aws/aws-sdk-go/service/rds/rdsutils`

Reply:
Good point. I would change the `if (selection.isSingleLine)` to `if (!selection.isEmpty)`.
There could be a package with a very long import path or someone with a very zoomed in window.